settings.h

```diff
@@ -714,6 +718,9 @@ typedef struct {
     float steps_per_mm;
     float max_rate;
     float acceleration;
+#if ENABLE_JERK_ACCELERATION   
+    float jerk;
+#endif
```

Comment:
This is another potential problem we have to be aware of, the area reserved for global settings may overflow when many axes are configured. I have already switched to claim settings storage for a second PWM spindle from above the area reseved for the core. Perhaps the jerk settings should as well? I'll have to think about that. See nvs.h.

Reply:
I have just [commited a version](https://github.com/grblHAL/core/blob/master/changelog.md#20241208) with updated settings structures and that adds a [permanent field for jerk](https://github.com/grblHAL/core/blob/a41790dada31aa0fc1c49b92fe4df033a9533079/settings.h#L715). I have also reserved ten new axis settings for the core, [jerk beeing the first assigned](https://github.com/grblHAL/core/blob/a41790dada31aa0fc1c49b92fe4df033a9533079/settings.h#L527). 

If you can update your PR against this version then I will be able to merge it without risking issues with the settings.

Reply:
Currently at work, will take a look and push latest revision tomorrow night or maybe wednesday! 
Thanks for all the work! :)

Reply:
resolved the conflicts. :)